openml/testing.py

```diff
@@ -223,6 +225,10 @@ def _check_fold_timing_evaluations(
             "wall_clock_time_millis_testing": (0, max_time_allowed),
             "wall_clock_time_millis": (0, max_time_allowed),
         }
+        if LooseVersion(sklearn.__version__) >= "0.23":
```

Comment:
Why is it these can't/don't need to get tested?

Reply:
This was a placeholder check since there was only [one unit test](https://github.com/openml/openml-python/blob/dcf2d22e4b92b6afc5bfb15b69c4f8d37f15d57a/tests/test_extensions/test_sklearn_extension/test_sklearn_extension.py#L1809) which had a slightly different failure point.

Looked into it and had to make [this change](https://github.com/openml/openml-python/blob/dcf2d22e4b92b6afc5bfb15b69c4f8d37f15d57a/openml/extensions/sklearn/extension.py#L1349) to account for [this change](https://github.com/scikit-learn/scikit-learn/blob/0.23.1/sklearn/cluster/_kmeans.py#L273) in K-means in scikit-learn 0.23.1

Not sure if this is the most scalable, long-standing solution.